docs/core/resilience/snippets/http-resilience/Program.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Net;
+using Microsoft.Extensions.DependencyInjection;
+using Polly;
+using Polly.CircuitBreaker;
+using Polly.Retry;
+
+var services = new ServiceCollection();
+
+var builder = services.AddHttpClient<ExampleClient>(
+    configureClient: static client =>
+    {
+        client.BaseAddress = new("https://jsonplaceholder.typicode.com");
+    });
+
+builder.AddStandardResilienceHandler(options =>
+{
+    options.RetryOptions.BackoffType = DelayBackoffType.Exponential;
+});
+
+builder.AddResilienceHandler("CustomPipeline", static builder =>
+{
+    // See: https://www.pollydocs.org/strategies/retry.html
+    builder.AddRetry(new RetryStrategyOptions<HttpResponseMessage>
```

Comment:
```suggestion
    builder.AddRetry(new HttpRetryStrategyOptions
```

better to use http-specific options here